### PR TITLE
feat(es-dev-server): add https redirect

### PR DESCRIPTION
**Pull request by ruphin**
Adds an HTTPS redirect when using es-dev-server in https mode.

### CODE REVIEW

File: packages/es-dev-server/src/create-server.js [hunk 1]
```diff
@@ -36,7 +47,46 @@ export function createServer(cfg, fileWatcher = chokidar.watch([])) {
       allowHTTP1: true,
```
Comment (LarsDenBakker) [inline packages/es-dev-server/src/create-server.js:47]: I was expecting this one to have to be false to allow connecting to the redirect server

**Comment by LarsDenBakker** [comment]
LGTM, there are some linting errors

File: packages/es-dev-server/src/create-server.js (hunk 1 above)
Comment (ruphin) [reply packages/es-dev-server/src/create-server.js:47]: The application server can keep accepting HTTP1 and HTTP2 connections like it was before.

The new TCP server accepts anything and just forwards requests to either the redirect server or the application server, the protocols that they accept have no impact on this.

**Comment by ruphin** [comment]
I am unable to run linting or tests in this project so I cannot fix any problems. Feel free to add commits to this MR, or just take the changes and apply them locally, and drop this MR.

**Comment by LarsDenBakker** [comment]
It should be a matter of `yarn install` and `yarn format` from the root of the project. Is that not working for you?

**Comment by ruphin** [comment]
`npm install` breaks for me at some point as it tries to run lerna things
`npm run format` breaks as it tries to load `eslint-config-prettier` which doesn't exist

I think it's easier for you to just run the linter and commit whatever changes you need.

**Comment by LarsDenBakker** [comment]
That's because we use yarn workspaces, so you need to use `yarn install` and `yarn format` :)

I can also do it when I get the time.

**Comment by ruphin** [comment]
I don't have yarn, best if you do it :)

**Comment by ruphin** [comment]
Sweet, thanks!

**Comment by bennypowers** [comment]
np
try `npx yarn format` next time

**Pull request merged**